src/starcheck.pl

```diff
@@ -578,6 +566,7 @@ sub json_obsids{
 
     $obs{$obsid}->check_monitor_commanding(\@bs, $or{$obsid});
     $obs{$obsid}->check_flick_pix_mon();
+    $obs{$obsid}->set_dynamic_mag_limits();
```

Comment:
I think that's a slippery slope that could apply to every change that affects the output format.  Does the code actually run OK on very old loads?  I think the previous PCAD alignment check update was a little different than this because of the underlying process and interface change that drove the new check.